.env

```diff
@@ -0,0 +1,6 @@
+# Database
+MYSQLDB_ROOT_PASSWORD=root
+MYSQLDB_DATABASE=db
+
+# App
+SPRING_PROFILES_ACTIVE=dev
```

Comment:
git rm --cached .env - .env should never be in version control, add .env.template instead with actual values removed